app/src/pages/started.tsx

```diff
@@ -0,0 +1,69 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Link from 'next/link'
+import FindTask from 'components/IconSvg/FindTask'
+import CreateTask from 'components/IconSvg/CreateTask'
+import FindTeam from 'components/IconSvg/FindTeam'
+
+const links = [
+  {
+    icon: FindTask,
+    title: 'Find a task',
+    subtitle: 'Create an account as organization',
+    buttonLink: ''
+  },
+  {
+    icon: CreateTask,
+    title: 'Create a task',
+    subtitle: 'Create an account as freelancer',
+    buttonLink: ''
+  },
+  {
+    icon: FindTeam,
+    title: 'Find my team',
+    subtitle: 'Find your organization',
+    buttonLink: ''
+  }
+]
+
+const Home: NextPage = () => {
+  return (
+    <div className='container'>
+      <Head>
+        <title>Getting Started</title>
+        <meta
+          name='description'
+          content='Buildstream for Freelancers and Companies'
+        />
+      </Head>
+
+      <main className='flex flex-col gap-28 flex-auto h-screen'>
+        <section className='grid-layout mt-28'>
+          <span className='col-span-8 block text-[#17191A] tracking-[-4px] text-[112px] leading-[111px] font-bold'>
+            <span className='block'>What you</span> wanna do first?
+          </span>
+        </section>
+        <section className='grid-layout'>
+          {links.map((link, index) => {
+            return (
+              <div className='paper col-span-4 flex flex-col gap-9'key={index} >
+                <link.icon />
+                <div className='flex flex-col items-start'>
+                  <span className='block font-bold text-4xl text-sm'>
+                    {link.title}
+                  </span>
+                  <span className='font-normal opacity-50'>
+                    {link.subtitle}
+                  </span>
+                </div>
+                <button className='btn-primary'>Get started now</button>
```

Comment:
We may want to connect wallet when they click create task if they have not connected already. it should route to `organization/create` when it's done